src/components v2/Molecules/Components/Accordion/Accordion.vue

```diff
@@ -97,7 +100,7 @@ export const props = {
     type: Boolean as PropType<boolean>,
     default: false,
   },
-  mobileNoPadding: {
+  noHorizontalPadding: {
     type: Boolean as PropType<boolean>,
     default: false,
   },
```

Comment:
**issue:** you can pull this padding prop from the `AccordionContentProps` on line 66.